src/main/waveform/waveform.core.js

```diff
@@ -175,6 +165,31 @@ define([
         peaks.on("user_scrub.*", function(time){
           peaks.player.seekBySeconds(time);
         });
+      },
+
+      /**
+       *  This is the method that is going to fire once the window is resized
+       */
+      onResize: function() {
+        var that = this;
+
+        that.ui.overview.hidden = true;
+        that.ui.zoom.hidden = true;
+
+        if (that.resizeTimeoutId) clearTimeout(that.resizeTimeoutId);
+        that.resizeTimeoutId = setTimeout(function(){
+          var w = that.ui.player.clientWidth;
+          var overviewWaveformData = that.origWaveformData.resample(w);
+          peaks.emit("resizeEndOverview", w, overviewWaveformData);
+          peaks.emit("window_resized", w, that.origWaveformData);
+        }, 500)
+      },
+
+      /**
+       * Destroy the even listener
+       */
+      destroy: function() {
+        window.removeEventListener("resize", this.resizeListener);
```

Comment:
I'd probably make sure this.resizeListener is defined. you might not have created it, if no data was loaded or loading remote data didn't work.